students/kaczmarek_katarzyna/lesson_06_dicts_tuples_sets_args_kwargs/uppercase.py

```diff
@@ -0,0 +1,19 @@
+def capitalize(lower_case_word):
+    ascii_code = ord(lower_case_word[0])
+    if 97 <= ascii_code <= 122:
```

Comment:
można również wykorzystać funkcję upper i zredukowac to do:
print(lower_case_word[0].upper() + lower_case_word[1:], end=" ")